crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -134,18 +134,22 @@ pub struct BuildSpecCommand {
 	/// [default: ./chain-spec.json].
 	#[arg(short = 'o', long = "output")]
 	pub(crate) output_file: Option<PathBuf>,
-	/// For production, always build in release mode to exclude debug features.
+	/// [DEPRECATED] For production, always build in release mode to exclude debug features.
 	#[clap(short = 'r', long, default_value = "true")]
 	pub(crate) release: bool,
 	/// Parachain ID to be used when generating the chain spec files.
 	#[arg(short = 'i', long = "id")]
 	pub(crate) id: Option<u32>,
 	/// Whether to keep localhost as a bootnode.
-	#[clap(long, default_value = "true")]
+	#[clap(long, default_value = "false")]
```

Comment:
```suggestion
	#[clap(long)]
```
Surely a default value for a bool is false?